pep/app/templates/head.hbs

```diff
@@ -7,9 +7,7 @@ setting values on the 'head-data' service.
     {{this.model.title}}
 </title>
 <meta name="description" content="A Psychoanalytic Library at your fingertips" />
-{{#if this.model.themePath}}
```

Comment:
i know there was a reason i had this wrapped in an if block when i first did this...but cant remember exactly why now. Maybe there were instances in runloops where the themePath was not defined yet, on boot or something, so the `<link>` would spew 404s?

Reply:
From what I can see, everything works great. Things are a bit different now - as this template only gets run in fastboot